### PR TITLE
Add GOVERNANCE and MAINTAINERS

### PR DESCRIPTION
**Pull request by mwestphal**

### CODE REVIEW

File: doc/dev/GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Governance Policy
+
+This document provides the governance policy for the F3D Project. Maintainers agree to this policy and to abide by all Project polices, including the [code of conduct](../../CODE-OF-CONDUCT.md) by adding their name to the [maintainers.md file](./MAINTAINERS.md).
+
+## 1. Roles.
+
+This project include the following roles.
```
Comment (nathanielcopeland) [inline doc/dev/GOVERNANCE.md:7]: includes

File: doc/dev/GOVERNANCE.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Governance Policy
+
+This document provides the governance policy for the F3D Project. Maintainers agree to this policy and to abide by all Project polices, including the [code of conduct](../../CODE-OF-CONDUCT.md) by adding their name to the [maintainers.md file](./MAINTAINERS.md).
```
Comment (nathanielcopeland) [inline doc/dev/GOVERNANCE.md:3]: I think 'project' should be lowercase, its used throughout the document with both capital P and lowercase p, so at a minimum it should at least be consistent

Comment (Meakk) [reply doc/dev/GOVERNANCE.md:3]: I would keep `Project` with an uppercase, but we need to define it when it appears the first time, just like other specific terms like `Maintainers` or `Contributors`. Something like:

```suggestion
This document provides the governance policy for F3D (the "Project"). [Every person](./MAINTAINERS.md) responsible for its maintenance (the "Maintainers") agrees to this policy and abides by all Project policies, including the [code of conduct](../../CODE-OF-CONDUCT.md) by adding their name to the [maintainers.md file](./MAINTAINERS.md).
```

File: doc/dev/GOVERNANCE.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## 1. Roles.
+
+This project include the following roles.
+
+**1.1. Maintainers**. Maintainers are responsible for organizing activities around developing, maintaining, and updating the Project. Maintainers are also responsible for determining consensus. This Project may add or remove Maintainers with the approval of the current Maintainers.
+
+**1.2. Contributors**. Contributors are those that have made contributions to the Project.
```
Comment (Meakk) [inline doc/dev/GOVERNANCE.md:11]:
```suggestion
**1.2. Contributors**. Contributors are those who have made contributions to the Project.
```

File: doc/dev/GOVERNANCE.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+## 2. Decisions.
+
+**2.1. Consensus-Based Decision Making**. Projects make decisions through consensus of the Maintainers. While explicit agreement of all Maintainers is preferred, it is not required for consensus. Rather, the Maintainers will determine consensus based on their good faith consideration of a number of factors, including the dominant view of the Contributors and nature of support and objections. The Maintainers will document evidence of consensus in accordance with these requirements using issues comments.
```
Comment (Meakk) [inline doc/dev/GOVERNANCE.md:15]:
```suggestion
**2.1. Consensus-Based Decision Making**. Projects make decisions through the consensus of the Maintainers. While explicit agreement of all Maintainers is preferred, it is not required for consensus. Rather, the Maintainers will determine consensus based on their good faith consideration of several factors, including the dominant view of the Contributors and the nature of support and objections. The Maintainers will document evidence of consensus in accordance with these requirements using issues comments.
```

File: README.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
...
+[![CI](https://img.shields.io/github/actions/workflow/status/f3d-app/f3d/ci.yml?label=CI&logo=github)](https://github.com/f3d-app/f3d/actions/workflows/ci.yml) [![Packaging](https://img.shields.io/github/actions/workflow/status/f3d-app/f3d-superbuild/nightly.yml?label=Packaging&logo=github)](https://github.com/f3d-app/f3d-superbuild) [![codecov](https://codecov.io/gh/f3d-app/f3d/branch/master/graph/badge.svg?token=siwG82IXK7)](https://codecov.io/gh/f3d-app/f3d) [![Downloads](https://img.shields.io/github/downloads/f3d-app/f3d/total.svg)](https://github.com/f3d-app/f3d/releases) [![Sponsors](https://img.shields.io/static/v1?label=Sponsor&message=%E2%9D%A4&logo=GitHub&color=%23fe8e86)](https://github.com/sponsors/f3d-app) [![Discord](https://discordapp.com/api/guilds/1046005690809978911/widget.png?style=shield)](https://discord.f3d.app) [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](code_of_conduct.md)
```
[the remark this answers is not in the document]
Comment (mwestphal) [reply README.md:1]: it is suggested on multiple website that some people would only contribute on project with a CoC, lets make it clear we have one.

**Comment by mwestphal** [comment]
please review again @nathanielcopeland @Meakk

**Comment by mwestphal** [comment]
period could be removed I agree.

The ** are just bold, not headers.

**Comment by snoyer** [comment]
> The ** are just bold, not headers.

Sure but aren't they semantically (sub-)headings? in which case they might as well be proper headings in the markup.

- pros: less hacky, can be anchor linked (eg [section 3](https://github.com/mwestphal/f3d/blob/governance/doc/dev/GOVERNANCE.md#3-how-we-work))
- cons: corresponding paragraphs are very short so they'd _look_ better as run-in headings? (which is what the MVG authors were trying to emulate I assume)

**Comment by mwestphal** [comment]
yeah I think that it looks better

**Comment by mwestphal** [comment]
I removed the periods.

File: doc/dev/GOVERNANCE.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
+# Governance Policy
+
+This document provides the governance policy for F3D (the "Project"). [Every person](./MAINTAINERS.md) responsible for its maintenance (the "Maintainers") agrees to this policy and to abide by all Project polices, including the [code of conduct](../../CODE-OF-CONDUCT.md) by adding their name to the [MAINTAINERS.md file](./MAINTAINERS.md).
+
+## 1. Roles
+
+This Project includes the following roles.
+
+**1.1. Maintainers**. Maintainers are responsible for organizing activities around developing, maintaining, and updating the Project. Maintainers are also responsible for determining consensus. The Project may add or remove Maintainers by affirmative vote of 2/3 of all Maintainers.
```
[the remark this answers is not in the document]
Comment (mwestphal) [reply doc/dev/GOVERNANCE.md:9]: that doesnt seems necessary to me, cause the section will be quite short

**Comment by mwestphal** [comment]
looks like @Meakk and @snoyer agree that it should be subsection, ill make the change.

File: doc/dev/GOVERNANCE.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+### 1.2. Contributors
+
+Contributors are those who have made contributions to the Project.
+
+## 2. Decisions
+
+### 2.1 Consensus-Based Decision Making
+
+The Project make decisions through the consensus of the Maintainers. While explicit agreement of all Maintainers is preferred, it is not required for consensus. Rather, the Maintainers will determine consensus based on their good faith consideration of a several factors, including the dominant view of the Contributors and the nature of support and objections. The Maintainers will document evidence of consensus in accordance with these requirements using issues comments.
```
Comment (Meakk) [inline doc/dev/GOVERNANCE.md:21]:
```suggestion
The Project makes decisions through the consensus of the Maintainers. While explicit agreement of all Maintainers is preferred, it is not required for consensus. Rather, the Maintainers will determine consensus based on their good faith consideration of several factors, including the dominant view of the Contributors and the nature of support and objections. The Maintainers will document evidence of consensus in accordance with these requirements using issues comments.
```

File: doc/dev/MAINTAINERS.md [hunk 8]
```diff
@@ -0,0 +1,11 @@
+# Maintainers
+
+This document lists the Maintainers of the Project. Maintainers may be added once approved by the existing maintainers as described in the [Governance document](./GOVERNANCE.md). By adding your name to this list you are agreeing to abide by the Project governance documents and to abide by all of the Organization's polices, including the [code of conduct](../../CODE_OF_CONDUCT.md).
```
Comment (Meakk) [inline doc/dev/MAINTAINERS.md:3]:
```suggestion
This document lists the Maintainers of the Project. Maintainers may be added once approved by the existing maintainers as described in the [Governance document](./GOVERNANCE.md). By adding your name to this list you are agreeing to abide by the Project governance documents and to abide by all of the Organization's policies, including the [code of conduct](../../CODE_OF_CONDUCT.md).
```

File: doc/dev/GOVERNANCE.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
+# Governance Policy
+
+This document provides the governance policy for F3D (the "Project"). [Every person](./MAINTAINERS.md) responsible for its maintenance (the "Maintainers") agrees to this policy and to abide by all Project polices, including the [code of conduct](../../CODE-OF-CONDUCT.md) by adding their name to the [MAINTAINERS.md file](./MAINTAINERS.md).
```
Comment (nathanielcopeland) [inline doc/dev/GOVERNANCE.md:3]:
```suggestion
This document provides the governance policy for F3D (the "Project"). [Every person](./MAINTAINERS.md) responsible for its maintenance (the "Maintainers") agrees to this policy and to abide by all Project policies, including the [code of conduct](../../CODE-OF-CONDUCT.md) by adding their name to the [MAINTAINERS.md file](./MAINTAINERS.md).
```

File: doc/dev/MAINTAINERS.md (hunk 8 above)
Comment (nathanielcopeland) [reply doc/dev/MAINTAINERS.md:3]:
```suggestion
This document lists the Maintainers of the Project. Maintainers may be added once approved by the existing Maintainers as described in the [Governance document](./GOVERNANCE.md). By adding your name to this list you are agreeing to abide by the Project governance documents and to abide by all of the Organization's policies, including the [code of conduct](../../CODE_OF_CONDUCT.md).
```

**Review by Meakk — APPROVED** [review]

**Pull request merged**